api/swagger.yml

```diff
@@ -59,9 +59,9 @@ externalDocs:
   description: Find out more about Hermez network.
   url: 'https://hermez.io'
 servers:
-  - description: Hosted mock up
-    url: https://apimock.hermez.network/v1
-  - description: Localhost mock Up
+  - description: Testnet api, connected to Rinkeby deployment
+    url: https://api.testnet.hermez.io/v1
```

Comment:
I'm not sure if this information should go here. I understand this swagger config file as the file to run the mock api locally, but not as the directory list of the current available deployed servers.
I think that Hosted mockup server should not go to this file, and should go to the documentation website (in fact, it already is there: https://docs.hermez.io/#/developers/api ).
In this way, if the public deployed servers list changes, there is no need to update this file (which is unrelated to a public list of servers, as is a swagger config file).
```suggestion
```

Reply:
After an off github chat, @arnaucube made me realise that there is a major problem with using testnet for Swagger playground: there are several endpoints that are not straight forward to work in a sense of achieving a `200` response.

This produces a tradeoff situation:

- Mock server: all responses will be correct, except the ones that violate a basic validation constrains defined in Swagger, this may lead to a misunderstanding on how the validation works in a real API, but will always return interesting examples.
- "Real" testnet server: input parameters will actually produce different results and validation will work as expected. But it may be difficult or very difficult to achieve `200` respnses in some endpoints